### ISSUE TITLE
electron: 'save as...' command not added to file menu

### ISSUE THREAD

**Issue by vince-fugnitto** [eclipse-theia/theia#9134]
#### Bug Description:

Since we do not support dynamic main-menu items in `electron` (https://github.com/eclipse-theia/theia/issues/446), the `save as...` command is never added to the `file` main-menu.

https://github.com/eclipse-theia/theia/blob/8d5e12accfbd6bbe0cd720d3ebd151b700f4dc80/packages/workspace/src/browser/workspace-frontend-contribution.ts#L152-L155

Is it something we should enable always?

#### Steps to Reproduce:

1. start the example-electron application with a workspace
2. open an editor
3. open the `file` main-menu - notice no `save as...`

**Comment by vince-fugnitto** [comment]
@kittaakos have you noticed this issue as well? I'm wondering if for electron we enable the `save as...` always so it is available in the _file_ main-menu.

**Comment by kittaakos** [comment]
> have you noticed this issue as well?

No, I haven't. I open the example electron app, and `File` > `Save as...` is there for me.

**Comment by vince-fugnitto** [comment]
> > have you noticed this issue as well?
> 
> No, I haven't. I open the example electron app, and `File` > `Save as...` is there for me.

Odd :( It happens for me on Linux at least, the enablement of the command is flaky at best:


https://user-images.githubusercontent.com/40359487/109672914-7f146380-7b43-11eb-9152-8685a72d58d6.mp4

**Comment by sgraband** [comment]
`Save As` unconditionally registered in `CommonMenus.FILE_SAVE`.
